cdap-ui/app/cdap/components/Footer/index.tsx

```diff
@@ -20,7 +20,48 @@ import If from '../If';
 import { isNilOrEmptyString, objectQuery } from 'services/helpers';
 import { getCurrentNamespace } from 'services/NamespaceStore';
 import NamespaceStore from 'services/NamespaceStore';
-require('./Footer.scss');
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    background: 'white',
+    color: '#cccccc',
+    fontSize: '11px',
```

Comment:
We shouldn't define font sizes for custom elements. Any specific reason why we are not inheriting from parent?